.github/workflows/tests.yml

```diff
@@ -21,6 +22,27 @@ jobs:
       cancel-in-progress: true
     steps:
       - uses: actions/checkout@v3
+        with:
+          lfs: 'true'
```

Comment:
`lfs: 'true'` was maybe the only thing which was missing to make the screenshot test not failing, WDYT?

Reply:
OK, so this is not enough, we have to explicitly invoke `verifyPaparazziDebug` task for the test to fail :/

Reply:
Yep. Also, in my case the tolerance had to be > `0.0` to work, as there are very tiny difference in maybe 6-8 pixels between my generated screenshots and the ones from the CI. It might be due to some color callibration of the OS? I'm not sure, to be honest, but they didn't match 100%.

Maybe something like `0.01` could work?